src/spikeinterface/core/baserecording.py

```diff
@@ -97,21 +94,21 @@ def list_to_string(lst, max_size=6):
         return txt
 
     def _repr_header(self):
-        class_name = self.__class__.__name__
-        name_to_display = class_name
         num_segments = self.get_num_segments()
         num_channels = self.get_num_channels()
-        sf_khz = self.get_sampling_frequency() / 1000.0
+        sf_hz = self.get_sampling_frequency()
+        sf_khz = sf_hz / 1000
         dtype = self.get_dtype()
 
         total_samples = self.get_total_samples()
         total_duration = self.get_total_duration()
         total_memory_size = self.get_total_memory_size()
+        sampling_frequency_repr = f"{sf_khz:0.1f}kHz" if sf_hz > 10000 else f"{sf_hz}Hz"
```

Comment:
```suggestion
        sampling_frequency_repr = f"{sf_khz:0.1f}kHz" if sf_hz > 10000 else f"{sf_hz:0.3f}Hz"
```

Good, I remember we mentioned this.

What about this small improvement?

Reply:
The problem was that for LFP data sampled at 2.5khz, the repr was printing `0.2kHz` (because of the 1 decimal).

On the other hand, if we forced 2 decimals, it would have been a bit weird to have `30.00kHz`.

So now it's going to print:
- `2500.0Hz`
- `30.0kHz`

which I think is nicer. I placed the "cutoff" at 10kHz

Reply:
Ah, OK, So yes,  I assumed wrongly that sometimes for LFP you will have sampling rates that would not be an integer so I wanted to contain the printing of arbitrary digits.

Do you think it would be good to add f"{sf_hz:0.1f}Hz" instead or is it OK that in the rare case where there are more decimals it gets printed?

Reply:
yea 0.1 is better. Can you change the suggestion?

Reply:
Commited it.